scrapers/SexVR.yml

```diff
@@ -0,0 +1,82 @@
+name: "SexVR"
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.sexvr.com/search/pornstars/{}/
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - sexvr.com/pornstars
+    scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sexvr.com/video
+    scraper: sceneScraper
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //div[@class="item-col item--pornstar col"]//a/@title
+      URL: //div[@class="item-col item--pornstar col"]//a/@href
+  performerScraper:
+    common:
+      $info: //div[@class="profile-info"]
+    performer:
+      Name: $info//li[@class="profile-field profile-field-name"]/span[@class="sub-desc"]/text()
+      Gender:
+        selector: //meta[@name="application-name"]/@content
+        replace:
+          - regex: .+
+            with: female
+      Twitter: $info//li[span[@class="sub-label"]/text()="Twitter:"]/span[@class="sub-desc"]/a/@href
+      Birthdate:
+        selector: $info//li[@class="profile-field profile-field-fob"]/span[@class="sub-desc"]/text()
+        replace:
+          - regex: \s\(\d+\s\w+\)
+            with:
+        parseDate: 2 Jan 2006
+      Ethnicity:
+        selector: $info//li[@class="profile-field profile-field-ethnicity"]/span[@class="sub-desc"]/text()
```

Comment:
on performers which have date of birth (`https://www.sexvr.com/pornstars/aletta-ocean-26.html`) populated we get 2 matches. 
switching to 
`selector: $info//li[@class="profile-field profile-field-ethnicity"]/span[text()="Ethnicity:"]/../span[@class="sub-desc"]/text()`
seems to work